adal/tests/Test.ADAL.NET.UIAutomation/XamarinDroidADALTests.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using NUnit.Framework;
+using Test.Microsoft.Identity.Core.UIAutomation;
+using Test.Microsoft.Identity.Core.UIAutomation.infrastructure;
+using Xamarin.UITest;
+using Xamarin.UITest.Queries;
+
+namespace Test.ADAL.NET.UIAutomation
+{
+    /// <summary>
+    /// Configures environment for core/android tests to run
+    /// </summary>
+    [TestFixture(Platform.Android)]
+    class XamarinDroidADALTests
+    {
+        IApp app;
+        Platform platform;
+        ITestController xamarinController;
+
+        public XamarinDroidADALTests(Platform platform)
+        {
+            this.platform = platform;
+        }
+
+        /// <summary>
+        /// Initializes test controller
+        /// </summary>
+        [SetUp]
+        public void BeforeEachTest()
+        {
+            app = AppFactory.StartApp(platform);
```

Comment:
>app [](start = 12, length = 3)

are you sure you want to re-create the app with-out setting it in the controller? This makes very little sense to me. Would be great if you can make the code reveal the intent

Reply:
this is a bug, resolved